### PR TITLE
Remove CE-only warning from shared tests

### PR DESCRIPTION
**Pull request by banks**
I added this while developing #25992 as a quick check that the warnings were omitted (and that this test functionality worked) but didn't intend to commit it since it will conflict with Enterprise code.

Perhaps later it would be nice to have a clean way to be able to encode these warnings here but only for CE but that is a bigger change that is a bit gross so want to just get this mergeable in Ent for now while we discuss the cleanest pattern for that.

### CODE REVIEW

**Comment by banks** [comment]
Changes incoming: I realise we already have a mechanism for this so will update the PR to use it.

File: physical/raft/config_test.go [hunk 1]
```diff
@@ -50,6 +58,7 @@ func TestRaft_ParseConfig(t *testing.T) {
 				cfg.RaftLogVerifierEnabled = true
 				cfg.RaftLogVerificationInterval = defaultRaftLogVerificationInterval
 			},
+			wantWarns: ceOnlyWarnings("raft_log_verification_interval is less than the minimum allowed"),
```
Comment (miagilepner) [inline physical/raft/config_test.go:61]: is this warning not present in enterprise?

Comment (banks) [reply physical/raft/config_test.go:61]: Oh great catch. Will remove this wrapper.

Comment (banks) [reply physical/raft/config_test.go]: Fixed in [5ddc1ea](https://github.com/hashicorp/vault/pull/26052/commits/5ddc1ea4c56283170e27055256f9399edfc7fcfa)

**Review by miagilepner — APPROVED** [review]

**Pull request merged**